site/content/en/docs/Guides/Client SDKs/unreal.md

```diff
@@ -34,9 +34,6 @@ Available settings:
 {{% feature expiryVersion="1.1.0" %}}
 - Agones Sidecar IP. (default: `http://localhost:59358`)
 {{% /feature %}}
-{{% feature publishVersion="1.1.0" %}}
-- Agones Sidecar IP. (default: `http://localhost:${AGONES_SDK_HTTP_PORT}`)
```

Comment:
Why is this removed?

Reply:
Because I've removed this as a setting in the unreal editor. The code no longer uses the setting (and the setting no longer exists) so it should not be documented. 

I've previously changed it to reference the environment variable (even though I hadn't updated the code) but as I looked at the code I realized it didn't make sense as a setting.